how_to_ask_for_help.html

```diff
@@ -0,0 +1,44 @@
+</head>
+<body>
+    <h1>How to get Coding Help</h1>  
+    <p>This is a guide on how to get help when you run into a problem while coding.</p>
+    <p>Most people are very keen to offer help if they can, but I have noticed a few problems people make that mean it is harder for others to help them. This might cause you to get frustrated, and not get the help you need.</p>
+
+    <ol>
+        <li>
+            <h2>Let us see your code</h2>
+            <p>We are probably going to want to see the code you have typed, and try changing things. Sometimes it is not clear, right away, what is wrong. We want to edit the code and add logs and alter things. </p>
+            <ul>
+                <li>Put the code in a format we can edit. Screenshots or photos of the screen will not work for this, as people would have to type out all of your code just to start helping.</li>
+                <li>Include all of your code. If you just put part of the code that you think is the problem, we cannot help if it turns out the problem is in another part of your code. This is a big issue in CSS, where other parts of your CSS file can change things everywhere.</li>
+                <li>If you change something, post the updated code. This helps people see if you have made error when you have made corrections.</li>
+                <li>Use sites like <a href="https://github.com/">Github</a> or <a href="https://jsfiddle.net/">jsFiddle</a> to post your code in a way that people can see and work with it.</li>
+            </ul>
+        </li>
+        <li>
+            <h2>Tell us what you have tried</h2>
+        </li>
+        <li>
+            <h2>Let us know where your task comes from</h2>
+            <p>Codecademy's discord server has rules against helping others with assigments for school. Just saying what project you are doing, if you are working through somthing on codecademy, freecodecamp, the odin project or other sites like this, lets peole know that you are not asking people to help you with your homework.</p>
+            <p>Link to the page if it is a codecademy lesson or project. Many of us have completed these projects, but it is difficult to navigate back to find our solutions. If you include a link to the excercise, we can easily bring up our answers and compare code to find errors.</p>
+        </li>
+        <li>
+            <h2>Be aware of the following super handy resources to catch simple errors</h2>
+            <p><a href="https://validator.w3.org/">W3C website validator</a>: putting your page in here will let you know about many simple errors, such as unclosed tags.</p>
+            <p><a href="https://text-compare.com/">Text Compare</a>: Works like the diff built into codecademy. When you cannot see where you have made an error, and you are sure it looks like an example. This tool is useful for seeing where easily missed mistakes may be present.</p>
+        </li>
+        <li>
+            <h2>Tell us your problem</h2>
```

Comment:
What do you think about moving this section up to be number 2?
Then it would be:

1. Let us see your code
2. Tell us your problem
3. Tell us what you have tried
4. Let us know where your task comes from
5. Handy resources

I think that might improve the flow of the headers a little, but if you think a different order makes sense I'm all ears.